email/parser.go

```diff
@@ -27,7 +27,9 @@ const (
 )
 
 var (
-	skylinkRE           = regexp.MustCompile("^.*://([a-z0-9]{55})|://.*\\..*/([a-zA-Z0-9-_]{46}).*$")
+	skylink64RE = regexp.MustCompile(".+?://.+?\\..+?/([a-zA-Z0-9-_]{46})")
+	skylink32RE = regexp.MustCompile(".+?://([a-zA-Z0-9-_]{55})")
```

Comment:
REQ: This new version of the regex changes the original behaviour because it doesn't allow extra text before the base32 skylink.

Additionally, do we need to ensure that we're looking at an URI (the `.+?://` part)?

Reply:
Also, I wouldn't call this change "minor". Anything that needs additional tests should be important enough not to be minor.

Reply:
I'll remove the `MINOR`. I added it because I created this PR right after merging another PR that updated the parser regex, the real meat of the change was in the one that's merged already, this was kind of like an afterthought.

It's not a terribly important change, apparently, the parser has never caught multiple skylinks on a single line.
This PR makes it so it catches those, but we just never encounter it in production so it's never been an issue.

Reply:
The URI thing is already in master, we want to completely get rid of false positives. Sometimes we encounter a random string/hash that looks like a skylink and is included in the report, the email body sometimes contains those, by requiring it's a URI we get rid of those.

Every email I run through the parser matches all that it needs to match without false positives.
It's Danger that requested the tweak, so this will make his life easier.